docs/src/ice_dynamics.md

```diff
@@ -105,22 +111,25 @@ resolve_overloads!(ice_dynamics1D, op1_res_rules_1D, op2_res_rules_1D)
 to_graphviz(ice_dynamics1D)
 ```
 
-# Define a mesh
+## Define a mesh
 
 We'll need a mesh to simulate on. Since this is a 1D mesh, we can go ahead and make one right now:
 
 ``` @example DEC
-# This is a 1D mesh, consisting of edges and vertices.
-s′ = EmbeddedDeltaSet1D{Bool, Point2D}()
+# This is an empty 1D mesh.
```

Comment:
The previous comment needs to go back

Reply:
But this next line is an empty 1D mesh.

Reply:
The comment after correctly describes the primal mesh.